elixir/apps/domain/lib/domain/tokens.ex

```diff
@@ -38,6 +39,13 @@ defmodule Domain.Tokens do
     end
   end
 
+  def all_active_browser_session_tokens! do
```

Comment:
TODO: add a WHERE clause to only fetch the ones that expire in next 15 minutes or so. No point to refresh everything all the time.